### PR TITLE
Add image of retina to our data

### PR DESCRIPTION
**Pull request by sciunto**
## Description

To expand our data a little. I wanted to include this image in my PR on polar transform (but I need time to work on it) and I prefer to integrate the image first. Can be useful in our demos and it's a +1 in our scientific images.

### CODE REVIEW

**Comment by JDWarner** [comment]
I think you'll need to add the image itself as well ;)

File: skimage/data/__init__.py [hunk 1]
```diff
@@ -319,6 +320,27 @@ def hubble_deep_field():
...
+    -----
+    This image was downloaded from
+    `wikimedia <https://commons.wikimedia.org/wiki/File:Fundus_photograph_of_normal_left_eye.jpg>`.
+    This file is made available under the Creative Commons CC0 1.0 Universal
+    Public Domain Dedication.
+
+    Returns
+    -------
+    retina : (1411, 1411, 3) uint8 ndarray
+        Retina image.
```
Comment (hmaarrfk) [inline skimage/data/__init__.py]: RGB?

Comment (sciunto) [reply skimage/data/__init__.py]: Good suggestion. I added that.

**Comment by stefanv** [comment]
I'd recommend adding the citation provided on the Wikimedia page. We should reward those who open their data this way! 

Häggström, Mikael (2014). "Medical gallery of Mikael Häggström 2014". WikiJournal of Medicine 1 (2). DOI:10.15347/wjm/2014.008. ISSN 2002-4436. Public Domain.

**Comment by sciunto** [comment]
Excellent. Thank you @stefanv  It's done!

**Comment by jni** [comment]
Thanks @sciunto!

**Pull request merged**